lib/protoc/version.rb

```diff
@@ -1,3 +1,6 @@
 module Protoc
-  VERSION = '2.6.1'
+  # The version of the protobuf library from which these protoc binaries were built
+  PROTOBUF_VERSION = '2.6.1'
+  # This gem's version
+  VERSION = PROTOBUF_VERSION + '.1'
```

Comment:
The gem version needs to increase if a new version is to be released, but it's using the same protobuf version as before.  I would have liked to keep them in sync, but there's not really any way to do that in this case.  In this case 2.6.1 will still work for most people.  This PR is really only relevant to people for whom the existing binaries didn't work.